src/types/Recipient.ts

```diff
@@ -36,19 +36,24 @@ export class NotificationRecipient extends Recipient {
     readonly sendToAdmin: boolean;
     readonly integrationId: UUID | undefined;
     readonly key: string;
+    readonly description: string | undefined;
 
     public constructor(integrationId: UUID | undefined, sendToAdmin: boolean) {
         let displayName = 'Users:';
+        let description = '';
         if (sendToAdmin) {
             displayName += ' Admins';
+            description += ' Organization administrators for your account';
```

Comment:
No need to concatenate in this case 
```suggestion
            description = 'Organization administrators for your account';
```